keck_vnc_launcher.py

```diff
@@ -464,15 +469,24 @@ def get_sessions_requested(self, args):
         #get sessions to open
         #todo: use const SESSION_NAMES here
         sessions = list()
-        if args.control0  is True: sessions.append('control0')
-        if args.control1  is True: sessions.append('control1')
-        if args.control2  is True: sessions.append('control2')
-        if args.telstatus is True: sessions.append('telstatus')
-        if args.analysis0 is True: sessions.append('analysis0')
-        if args.analysis1 is True: sessions.append('analysis1')
-        if args.analysis2 is True: sessions.append('analysis2')
-        if args.telanalys is True: sessions.append('telanalys')
-        if args.status    is True: sessions.append('status')
+        if args.control0 is True:
```

Comment:
Good example of where I prefer readability to non-compliance.

Reply:
In general I find statements in-line with conditions to be less readable than having them broken up. I'm happy to let the PEP decide, but that's hardly generous on my part since it sounds like I agree with it!

Reply:
LOL.  I'm half way in between you two. :) I'm ok with a single line having `if condition: do_thing`, but I'm against the column aligning of it.

Reply:
I went and looked, because I didn't know for sure, and PEP 8 does discourage the use of compound statements of this type, it's a soft statement. I'd even go so far as to say it waffles a bit, saying it'd "rather not" have statements like 'if foo == 'blah': do_blah_thing()', but then it says "sometimes it's okay to put an if/for/while with a small body on the same line."

The extra whitespace to align does need to go. I used to do that, and then I realized it interferes with your ability to search through the code. If you still want the single-line compound statements I can put them back on the agree-to-disagree basis.

Reply:
I made this moot by addressing the "todo" in the comment above, iterating through SESSION_NAMES programatically instead of laying it all out explicitly.

Reply:
excellent thx, i didn't like these being hard coded down in the guts of the code.